internal/document/editor/editor.go

```diff
@@ -37,10 +37,29 @@ func Deserialize(data []byte) (*Notebook, error) {
 		PrefixAttributeName(InternalAttributePrefix, constants.FinalLineBreaksKey): fmt.Sprint(finalLinesBreaks),
 	}
 
-	f, info := document.ParseFrontmatter(string(sections.FrontMatter))
+	// If Front Matter exists, store it in Notebook's metadata.
+	if len(sections.FrontMatter) > 0 {
+		notebook.Metadata[PrefixAttributeName(InternalAttributePrefix, FrontmatterKey)] = string(sections.FrontMatter)
+	}
+
+	return notebook, nil
+}
+
+func DeserializeV2(data []byte, requireIdentity bool) (*Notebook, error) {
```

Comment:
@pastuxso can we consolidate `Deserialize` & `DeserializeV2`, the `fmt` won't have flags to toggle doc/cell identity options (default `false` for now). We can spin off a ticket to add those later.